R/task.R

```diff
@@ -182,6 +241,7 @@ task_eval <- function(id, envir = .GlobalEnv, verbose = FALSE, root = NULL) {
       data$type,
       explicit = task_eval_explicit(data, envir, verbose, root),
       expression = task_eval_expression(data, envir, verbose, root),
+      script = task_eval_script(data, envir, verbose, root),
```

Comment:
Do any of these three calls need `root` ?

Reply:
seems like not, probably they used to? I can remove that easily enough